packages/core/src/api/nodeConversions/nodeToBlock.ts

```diff
@@ -130,6 +131,11 @@ export function contentNodeToInlineContent<
       } else {
         const config = styleSchema[mark.type.name];
         if (!config) {
+          if (mark.type.spec.group?.includes("blocknoteIgnore")) {
```

Comment:
Could we store something else in the spec rather than using the group for this?

Or maybe by the mark's name?

This feels kind of wrong to use the group for this

Reply:
fair. Prefer just to add anything else on the spec? (i.e.: `(mark.type.spec as any).blocknoteIgnore = true`)?

Reply:
Yea, looks like the type should be already be open to allowing that sort of modifcation:

```
    /**
    Mark specs can include additional properties that can be
    inspected through [`MarkType.spec`](https://prosemirror.net/docs/ref/#model.MarkType.spec) when
    working with the mark.
    */
    [key: string]: any;
}
```

Reply:
nice, and it can be added in tiptap with:

extendMarkSchema(extension) {
    if (extension.name === "comment") { // if "comment" is the name of the comment mark
      return {
        blocknoteIgnore: true,
      };
    }
    return {};
  },

Reply:
Nice, much cleaner!